x-pack/test/apm_api_integration/tests/dependencies/metadata.ts

```diff
@@ -48,12 +48,15 @@ export default function ApiTest({ getService }: FtrProviderContext) {
     () => {
       it('returns correct metadata for the dependency', async () => {
         await generateData({ synthtraceEsClient, start, end });
+
         const { status, body } = await callApi();
         const { span } = dataConfig;
 
         expect(status).to.be(200);
         expect(body.metadata.spanType).to.equal(span.type);
         expect(body.metadata.spanSubtype).to.equal(span.subType);
+
+        synthtraceEsClient.clean();
```

Comment:
Don't you need to await this, to avoid the test ending before cleanup?
```suggestion
        await synthtraceEsClient.clean();
```

Reply:
Thanks, added the change in